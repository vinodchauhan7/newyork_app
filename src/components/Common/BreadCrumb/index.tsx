import React from 'react';

type Props = {};

const BreadCrumb = (props: Props) => {
    return (
        <div className="flex py-5 items-center">
            <div className="flex-grow border-t-2 border-black"></div>
                <span className="flex-shrink mx-4 text-1xl md:text-2xl text-blue-950">Most Popular Articles</span>
            <div className="flex-grow border-t-2 border-black"></div>
        </div>
    );
}

export default BreadCrumb;