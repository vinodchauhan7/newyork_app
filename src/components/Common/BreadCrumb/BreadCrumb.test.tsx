
import BreadCrumb from "./index"
import { render, screen} from "@testing-library/react";
import '@testing-library/jest-dom';

test("Render BreadCrumb Component", () => {
  const {container} = render(<BreadCrumb/>);
  const parent = screen.getByText(/Most Popular Articles/i);
  expect(container).toMatchSnapshot();
  expect(parent).toBeInTheDocument();
})