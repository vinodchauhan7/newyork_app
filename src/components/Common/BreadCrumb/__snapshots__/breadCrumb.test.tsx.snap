// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render BreadCrumb Component 1`] = `
<div>
  <div
    class="flex py-5 items-center"
  >
    <div
      class="flex-grow border-t-2 border-black"
    />
    <span
      class="flex-shrink mx-4 text-1xl md:text-2xl text-blue-950"
    >
      Most Popular Articles
    </span>
    <div
      class="flex-grow border-t-2 border-black"
    />
  </div>
</div>
`;
